Build/phpstan/phpstan-baseline.neon

```diff
@@ -1,2 +1,5 @@
 parameters:
-	ignoreErrors: []
+	ignoreErrors:
+		-
+			message: '#^Call to static method PHPUnit\\Framework\\Assert\:\:assertInstanceOf\(\) .* will always evaluate to#'
```

Comment:
This configuration is for something that we want to be excluded. So it should go into `phpstan.neon`, not into the baseline.

The baseline must not contain manual entries as it's expected to be generated by `composer phpstan:baseline`, and manual entries would get overwritten the next time someone regenerates the baseline.